### ISSUE TITLE
Reconcile cloudinary and the like button

### ISSUE THREAD

**Issue by laylar** [Eazy-Sneezy/Testing#44]
For some reason, adding Cloudinary broke the like button. Gonna figure out why and how to fix it.

**Comment by laylar** [comment]
I used "git checkout <commit>" to return to the last commit before we added Cloudinary, then I created a new branch from that called "cloudinary-like-patch". Now I'm going to use this blog post as a reference to try to re-add Cloudinary and see if it breaks again.

If it does, I'll try something else!

**Comment by laylar** [comment]
Aha! I've figured out what broke it.
![Screen Shot 2021-05-11 at 3 23 25 PM](https://user-images.githubusercontent.com/10251025/117892072-e437b400-b26c-11eb-90ba-4dbeb99c9cdb.png)

When we commented out 
`const { ObjectId } = require("bson");`
we lost the ability to parse ObjectIds, which is exactly the error we were getting.

When I uncommented it, everything works again. Except of course the feed's like button, but we've never gotten that to work yet, haha.
